src/back/services/events.ts

```diff
@@ -152,4 +159,58 @@ export class EventsService {
 
     return profiles
   }
+
+  static async commented(eventId: string | undefined, event: string | undefined, discoursePost: DiscourseWebhookPost) {
+    try {
+      if (
+        !eventId ||
+        !event ||
+        event !== 'post_created' || //TODO: we don't register editions or deletions
+        (await EventModel.alreadyRegistered(eventId)) ||
+        discoursePost.category_id !== DiscourseService.getCategory()
+      ) {
+        return
+      }
+
+      const commentedProposal = await ProposalModel.findOne<ProposalAttributes>({
+        discourse_topic_id: discoursePost.topic_id,
+      })
+      if (!commentedProposal) {
+        ErrorService.report('Unable to find commented proposal', {
+          discourse_data: {
+            event_id: eventId,
+            event,
+            discourse_post: discoursePost,
+          },
+          category: ErrorCategory.Events,
+        })
+        return
+      }
+
+      const user = await UserModel.findOne<UserAttributes>({ forum_id: discoursePost.user_id })
+
+      //TODO: for the time being we only register events for linked users comments
+      if (!user) {
+        return
+      }
+
+      const commentedEvent: CommentedEvent = {
+        id: crypto.randomUUID(),
+        address: user.address,
+        event_type: EventType.Commented,
+        event_data: {
+          discourse_event_id: eventId,
+          discourse_event: event,
+          discourse_post: discoursePost,
+          proposal_id: commentedProposal.id,
+          proposal_title: commentedProposal.title,
+        },
+        created_at: new Date(), //TODO: should we use the discord event creation date?
```

Comment:
If we have it in the discoursePost object let's do it. Maybe we should do that for votes and proposal too 🤔

Reply:
Not sure now... haha